ios/FluentUI/Command Bar/CommandBarItem.swift

```diff
@@ -18,7 +18,8 @@ open class CommandBarItem: NSObject {
         isEnabled: Bool = true,
         isSelected: Bool = false,
         itemTappedHandler: @escaping ItemTappedHandler = defaultItemTappedHandler,
-        accessbilityLabel: String? = nil
```

Comment:
Typo in existing API. Shall I go ahead and fix it too?

Reply:
Yes, please!
Thank you for fixing it!

Reply:
Fixed typo :) Thanks!